src/compass/utils/geometry_utils.py

```diff
@@ -265,3 +265,163 @@ def get_unit_vector4component_of_interest(los_inc_angle, los_az_angle, comp='enu
         ]
 
     return unit_vec
+
+
```

Comment:
A comment more on the `geometry_utils.py` module which might be for @vbrancat 
```
$ grep "def " src/compass/utils/geometry_utils.py | cut -d ' ' -f2 | cut -d'(' -f1  # list of functions defined here
los2orbit_azimuth_angle
azimuth2heading_angle
heading2azimuth_angle
enu2los
en2az
calc_azimuth_from_east_north_obs
get_unit_vector4component_of_interest
enu2rgaz
get_enu_vector_ecef
```

of these 9 fucntions, we're not using 5 of them, at least in the current state of this PR:




- azimuth2heading_angle
- enu2los
- en2az
- calc_azimuth_from_east_north_obs
- get_unit_vector4component_of_interest

Should we get rid of them until we're actually needing the function?

(how i checked they're not used:)
<details>

```bash
COMPASS$ grep "def " src/compass/utils/geometry_utils.py | cut -d ' ' -f2 | cut -d'(' -f1 | xargs -n1 git grep
src/compass/utils/geometry_utils.py:def los2orbit_azimuth_angle(los_az_angle, look_direction='right'):
src/compass/utils/geometry_utils.py:        orb_az_angle = los2orbit_azimuth_angle(los_az_angle)
src/compass/utils/geometry_utils.py:def azimuth2heading_angle(az_angle, look_direction='right'):
src/compass/utils/geometry_utils.py:def heading2azimuth_angle(head_angle, look_direction='right'):
src/compass/utils/geometry_utils.py:            az_angle = heading2azimuth_angle(head_angle)
src/compass/utils/geometry_utils.py:def enu2los(v_e, v_n, v_u, inc_angle, head_angle=None, az_angle=None):
src/compass/utils/geometry_utils.py:def get_unit_vector4component_of_interest(los_inc_angle, los_az_angle, comp='enu2los', horz_az_angle=None):
src/compass/utils/geometry_utils.py:        enu2los, en2los, hz2los, u2los, up2los, orb(it)_az, vert, horz
src/compass/utils/geometry_utils.py:        'enu2los', 'en2los', 'hz2los', 'horz2los', 'u2los', 'vert2los',   # radar LOS / cross-track
src/compass/utils/geometry_utils.py:    if comp in ['enu2los']:
src/compass/utils/geometry_utils.py:def en2az(v_e, v_n, orb_az_angle):
src/compass/utils/geometry_utils.py:        'en2az', 'hz2az', 'orb_az', 'orbit_az',                           # radar azimuth / along-track
src/compass/utils/geometry_utils.py:    elif comp in ['en2az', 'hz2az', 'orb_az', 'orbit_az']:
src/compass/utils/geometry_utils.py:def calc_azimuth_from_east_north_obs(east, north):
src/compass/utils/geometry_utils.py:def get_unit_vector4component_of_interest(los_inc_angle, los_az_angle, comp='enu2los', horz_az_angle=None):
src/compass/utils/geometry_utils.py:def enu2rgaz(radargrid_ref, orbit, ellipsoid,
src/compass/utils/lut.py:from compass.utils.geometry_utils import enu2rgaz
src/compass/utils/lut.py:    set_rg, set_az = enu2rgaz(burst.as_isce3_radargrid(), burst.orbit, ellipsoid,
src/compass/utils/geometry_utils.py:        vec_e, vec_n, vec_u = get_enu_vector_ecef(lon_deg, lat_deg)
src/compass/utils/geometry_utils.py:def get_enu_vector_ecef(lon, lat, units='degrees'):
```
</details>

Reply:
I've intentionally left the functions used for previous implementation about the conversion, which is based on radar geometry. Maybe it's a good idea to take off the unused functions before getting this PR merged. What do you think @vbrancat?

Reply:
My preference is to not remove them. We want to keep some functionalities down the line they might be useful. Anyhow, which of these functions are not used?

Reply:
> My preference is to not remove them. We want to keep some functionalities down the line they might be useful. 
Sounds good to me. I will keep those functions then.

> Anyhow, which of these functions are not used?
I meant the 5 functions that @scottstanie has mentioned above, and also was thinking about checking the codes to see if there are more.